auth_admin_passkey/model/res_users.py

```diff
@@ -116,22 +101,24 @@ def authenticate(self, db, login, password, user_agent_env):
                     self._send_email_passkey(cr, user_id, user_agent_env)
                 else:
                     self._send_email_same_password(cr, login)
-                cr.commit()
+                # we need to commit as we need to create the mails
+                # and we are not in a method with standard cursor management
+                cr.commit()  # pylint: disable=invalid-commit
```

Comment:
I think there is an argument in send_mail called `autocommit` or similar that makes the same.